mlir/lib/Mitigation/IR/MitigationOps.cpp

```diff
@@ -14,18 +14,20 @@
 
 #include "mlir/IR/Builders.h"
 #include "mlir/IR/OpImplementation.h"
+#include "llvm/ADT/TypeSwitch.h"
 
 #include "mlir/Dialect/Func/IR/FuncOps.h"
 
 #include "Mitigation/IR/MitigationDialect.h"
 #include "Mitigation/IR/MitigationOps.h"
 
-#define GET_OP_CLASSES
-#include "Mitigation/IR/MitigationOps.cpp.inc"
-
 using namespace mlir;
 using namespace catalyst::mitigation;
 
+#include "Mitigation/IR/MitigationEnums.cpp.inc"
+#define GET_OP_CLASSES
+#include "Mitigation/IR/MitigationOps.cpp.inc"
+
```

Comment:
Why were these moved down? Not a big deal but a bit unusual since the files should normally be generated without assuming particular namespace commands.

Reply:
I moved them down for coherence with https://github.com/PennyLaneAI/catalyst/blob/main/mlir/lib/Quantum/IR/QuantumOps.cpp#L30-L32